addEntry.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+require_once 'src/database/Database.php';
+
+require_once 'src/entries/EntriesModel.php';
+require_once 'src/languages/LanguagesModel.php';
+require_once 'src/snippets/SnippetsModel.php';
+
+// Form submission processing
+if(isset($_POST['title'])) {
+    $title = $_POST['title'];
+    $description = $_POST['description'];
+
+    $db = connectToDb();
+
+    $entriesModel = new EntriesModel($db);
+    $entrySuccess = $entriesModel->addNewEntry($db, $title, $description);
```

Comment:
As per my comment on the model, no need to pass $db in here.

Reply:
Fixed!